### PR TITLE
Add installation instructions (terraform/pip) to readme

### PR DESCRIPTION
**Pull request by chmreid**
Add installation instructions to readme. Covers both terraform and pip installation process.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,16 +2,52 @@
...
 1. set the secret with the tenant information in the aws secret-store [Secret Setup](#Secret-Setup)
 1. deploy infra -> sets up all the stuff. more here later. 
 
 
+### Terraform Installation
+
+In order to deploy the infra in this repository, you will need to install Terraform and an Auth0 Terraform
+provider. We use this Auth0 terraform provider: [alexkappa/terraform-provider-auth0](https://github.com/alexkappa/terraform-provider-auth0).
+
+To install terraform, download the Linux binary, unzip it to a directory on your path, and test it is available:
```
Comment (amarjandu) [inline README.md]: can osx users use the linux binary? i downloaded the Darwin ones.... 🤔

**Review by amarjandu — CHANGES_REQUESTED** [review]
just a small thing about OS binarys

File: README.md (hunk 1 above)
Comment (chmreid) [reply README.md]: Yeah, I was gonna call out Linux and Mac separately but I figured `s/linux/darwin/` was easy enough that I could leave it out. I'll add a note.

Comment (chmreid) [reply README.md]: I added a nice little trick

```
export OS=$(uname -s | tr '[:upper:]' '[:lower:]')
```

to get `linux` on Linux and `darwin` on Mac.

Comment (chmreid) [reply README.md]: Resolved with 75848e4

File: README.md [hunk 2]
```diff
@@ -2,16 +2,54 @@
...
+### Terraform Installation
+
+In order to deploy the infra in this repository, you will need to install Terraform and an Auth0 Terraform
+provider. We use this Auth0 terraform provider: [alexkappa/terraform-provider-auth0](https://github.com/alexkappa/terraform-provider-auth0).
+
+To install terraform, download the binary for your OS, unzip it to a directory on your path, and test it is available:
+
+```
+export TF_VERSION="0.12.16"
+export OS=$(uname -s | tr '[:upper:]' '[:lower:]')
```
Comment (amarjandu) [inline README.md:21]: dude 2 pro.

**Review by amarjandu — APPROVED** [review]
<3

**Pull request merged**